requests_toolbelt/adapters/ssl.py

```diff
@@ -48,3 +54,7 @@ def init_poolmanager(self, connections, maxsize, block=False):
             maxsize=maxsize,
             block=block,
             ssl_version=self.ssl_version)
+
+    def proxy_manager_for(self, *args, **kwargs):
+        kwargs['ssl_version'] = self.ssl_version
```

Comment:
So we presently have https://github.com/sigmavirus24/requests-toolbelt/blob/master/requests_toolbelt/adapters/socket_options.py#L49 in the socket options adapter.

This method *will* cause problems on anything <2.4.0, right? Since it is *possible* that people are already using  this with a version older than 2.4.0, we should endeavor not to break them. 

If I misunderstood your comment about skipping older versions of requests than this can be safely ignored probably.